compiler/passes/filesToAST.cpp

```diff
@@ -220,3 +239,49 @@ static void gatherWellKnownTypes() {
     }
   }
 }
+
```

Comment:
Why are some of your variables named with the preferred camelCase while others, apparently the local static ones, used underscores?  And why are they static?